src/App.js

```diff
@@ -6,13 +6,17 @@ import AddItem from './components/AddItem';
 import List from './components/List';
 import Welcome from './components/Welcome';
 import getToken from './lib/tokens';
+import { db } from './lib/firebase';
 
 function App() {
   const [token, setToken] = useState(localStorage.getItem('userToken') || '');
 
   const handleClick = () => {
     const token = getToken();
     localStorage.setItem('userToken', token);
+    db.collection('userTokens').add({
+      tokenName: token,
+    });
```

Comment:
This new code will add the newly generated token to a collection in firebase that is just a list of all existing tokens. This makes it easier to check if a list exists when a user is trying to join an existing list.

Reply:
Nice.